SystemInfo/EzcSystemInfoWrapper.php

```diff
@@ -0,0 +1,72 @@
+<?php
+
+/**
+ * File containing the EzcSystemInfoWrapper class.
+ *
+ * @copyright Copyright (C) eZ Systems AS. All rights reserved.
+ * @license For full copyright and license information view LICENSE file distributed with this source code.
+ */
+namespace EzSystems\EzSupportToolsBundle\SystemInfo;
+
+use ezcSystemInfo;
+
+/**
+ * This wraps zetacomponents/sysinfo, exposing its magic properties as public properties.
+ * Used here to allow lazy loading.
+ */
+class EzcSystemInfoWrapper
+{
+    /** @var string */
+    public $osType;
+
+    /** @var string */
+    public $osName;
+
+    /** @var string */
+    public $fileSystemType;
+
+    /** @var integer */
+    public $cpuCount;
+
+    /** @var string */
+    public $cpuType;
+
+    /** @var float */
+    public $cpuSpeed;
+
+    /** @var integer */
+    public $memorySize;
+
+    /** @var string */
+    public $lineSeparator;
+
+    /** @var string */
+    public $backupFileName;
+
+    /** @var array */
+    public $phpVersion;
+
+    /** @var \ezcSystemInfoAccelerator */
+    public $phpAccelerator;
+
+    /** @var bool */
+    public $isShellExecution;
+
+    public function __construct()
+    {
+        $ezcSystemInfo = ezcSystemInfo::getInstance();
```

Comment:
Could you please also take care about the exception while on it, as the upstream pull is taking a long time, ref:
```php
        try {
            $ezcSystemInfo = ezcSystemInfo::getInstance();
        } catch(ezcSystemInfoReaderCantScanOSException $e) {
            // Leave properties as null: https://github.com/zetacomponents/SystemInformation/pull/9
            return;
        }
```